charts/rustrial-aws-eks-iam-auth-controller/values.yaml

```diff
@@ -67,3 +67,17 @@ tolerations: []
 affinity: {}
 
 topologySpreadConstraints: {}
+
+defaultIdentity:
+  enabled: false
+  roleArn: "arn:aws:iam::123456789012:role/example-role"
+
+additionalIdentities: {}
```

Comment:
This is an array not an object and thus its default value must be `[]` and not `{}`.